codegen/arrai/grpc_client.arrai

```diff
@@ -33,11 +31,16 @@ let go = //{./go};
         }
 
         // NewClient creates a new Client.
-        func NewClient(addr string, connTimeout time.Duration) (*Client, error) {
+        func NewClient(addr string, connTimeout time.Duration, insecure bool) (*Client, error) {
```

Comment:
Shouldn't this be determined if the addr starts with `http` or `https`?

Reply:
I dont think gRPC dialing supports uri schemes.